client/src/components/content/ModelerSidebar/PropertiesPanel/PropertiesPanelComponents/PPParameterInput.jsx

```diff
@@ -57,33 +66,54 @@ const PPParameterInput = ({
     );
     setParameterValue(event.target.value);
   };
+
+  /**
+   * @description returns the Loch/Unlock-Icon on whether the Input-field is locked or unlocked
+   * @returns the corresponding icon
+   */
+  const returnLockIcon = () => {
+    if (userInputRequired) {
+      return (
+        <LockOutlined onClick={(event) => {
+          changeUserInputRequirement(event, variableName)
+        }} />
+      )
+    }
+    return (
+      <UnlockOutlined onClick={(event) => {
+        changeUserInputRequirement(event, variableName)
+      }} />
+    )
+  }
+
+  /**
+   * @returns the input value depending on whether the field is locked or unlocked
+   */
+  const returnPlaceholderValue = () => (
+    userInputRequired ? 'Will be specified at bot start' : 'Please type in value')
+
   return (
     <>
       <Text className={styles[`label-on-dark-background`]}>{variableName}</Text>
+      {isRequired && (
+        <Tooltip title='This field is required' className={styles.requiredStar}>
+          &nbsp;*
+        </Tooltip>
+      )}
+
       <Input
-        placeholder={variableName}
+        placeholder={returnPlaceholderValue()}
         defaultValue={value}
         value={userInputRequired ? '' : parameterValue}
-        onChange={(event) => changeParamterValue(event, variableName)}
-        onPressEnter={onValueChange}
+        onChange={(event) => changeParameterValue(event, variableName)}
+        onPressEnter={(event) => onValueChange(event, variableName)}
         suffix={
-          isRequired && (
-            <Tooltip title='This field is required for the task to work'>
-              <InfoCircleOutlined
-                style={{ color: corporateDesign.colorBackgroundCta }}
-              />
-            </Tooltip>
-          )
-        }
+          <Tooltip title={infoText}>
+            <InfoCircleOutlined />
+          </Tooltip>}
+        addonAfter={returnLockIcon()}
         disabled={userInputRequired}
       />
-      <Checkbox
-        onChange={(event) => changeUserInputRequirement(event, variableName)}
-        checked={userInputRequired}
-        className={styles[`label-on-dark-background`]}
-      >
-        User Input required
-      </Checkbox>
```

Comment:
The new lock looks awesome but from usability perspective it is not clear what this lock is for. Please figure out how we can still tell the user that by clicking that lock this parameter now requires user input. Perhaps it could also show a text when hovering over it?

Reply:
> In PPParameterInput the function changeUserInputRequirement is failing. You changed it compared to the DEV branch. On the DEv branch it still works. The problem is that the paramter property "requireUserInput" doesn't get set in the session Storage and therefore the requireUserInput state of the parameters is not persisted

Hey @WolfgangDaniel, I have also noticed this bug. the fixing was unfortunately not so easy. but now everything should work. I hope you are happy with the new design of the sidebar.
Feel free to look over and give feedback :) 
@kej-jay

Reply:
Just see my comment at the end of this page.